shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Utxo.hs

```diff
@@ -301,11 +304,18 @@ utxoInductive = do
   -- process Protocol Parameter Update Proposals
   ppup' <- trans @(PPUP crypto) $ TRC (PPUPEnv slot pp genDelegs, ppup, txup tx)
 
-  let outputs = Set.toList (eval (rng (txouts txb)))
+  -- TODO check voper
+  let outputs = Map.elems $ unUTxO (txouts txb)
       minUTxOValue = _minUTxOValue pp
-      outputsTooSmall = [out | out@(TxOut _ c) <- outputs, c < minUTxOValue]
+      outputsTooSmall = [out | out@(TxOut _ vl) <- outputs, (voper Gt) (vinject $ scaleVl vl minUTxOValue) vl]
   null outputsTooSmall ?! OutputTooSmallUTxO outputsTooSmall
 
+  -- TODO add forge
+  -- TODO add forge errors
+  -- let (Value vls) = _forge txb
+  -- let cids = Map.keys vls
+  -- all (adaID /=) cids  ?! (ForgingAda (Value vls))
```

Comment:
let's remove this and make an issue instead

Reply:
This is more to do with actually adding forging functionality than just doing the parametrization. Can definitely be removed from here.